### DISCUSSION TITLE
Explicit DataContext reference is a Visual Studio issue?

### DISCUSSION THREAD

**Discussion by sigmarsson** [PrismLibrary/Prism#2478] [Uno Platform] [answered]
The guys requested automatic view model wiring in the view beneath. Why do they need to have an explicit property then (in the code behind) to reference the `DataContext` and hence the respective view model again?

[Uno Sample Prism View](https://github.com/unoplatform/Uno.Samples/blob/master/UI/UnoContoso/UnoContoso.Shared/Views/OrderListView.xaml)

**Comment by brianlagunas** [comment, marked as answer]
They are using x:Bind which requires a strongly typed property in the code-behind
